src/intl/en.json

```diff
@@ -129,6 +129,29 @@
   "nav-use-aria-label": "Use Ethereum menu",
   "nav-learn-aria-label": "Learn menu",
   "on-this-page": "On this page",
+  "comp-tutorial-metadata-tip-author": "TIP AUTHOR",
```

Comment:
We should probably format this text to uppercase vs. storing here as uppercase.

Reply:
Yeah not a bad thought... I bet there are some others in there that are duplicated only to change the case that could potentially be styled to reduce redundancies. Let me know, I could post it as an issue if you want and take a look